.coveragerc

```diff
@@ -1078,6 +1078,7 @@ omit =
     homeassistant/components/saj/sensor.py
     homeassistant/components/satel_integra/*
     homeassistant/components/schluter/*
+    homeassistant/components/scrape/config_flow.py
```

Comment:
I don't think it's a good idea to add config_flow to the coverage exclusion.
It might be forgotten when we implement config-flow again.

Reply:
Didn't see much off an option than this

Reply:
Just leave the `.coveragerc` file as it was.
It's better to see the coverage failures, and have an explaination.

Coverage failures will not prevent this PR from going through